bin/compile

```diff
@@ -441,20 +443,46 @@ export GOPATH
 # GB installation
 mcount "pkgmanagement.$TOOL"
 
+mainPackagesInModule() {
+  go list -find -mod=vendor -f '{{ if eq .Name "main" }} {{.ImportPath}} {{ end }}' ./...
```

Comment:
Well that is nifty 😮 

This is a bit magic though. Can you add some comments about how this works for anyone who comes across it? Maybe a link to https://dave.cheney.net/2014/09/14/go-list-your-swiss-army-knife ?